src/app/Contact/Contact.module.css

```diff
@@ -66,7 +66,8 @@
 }
 
 .input:focus {
-  border-bottom-color: #3b82f6;
+  border-image: linear-gradient(90deg, #ec407a, #3b82f6) 1;
```

Comment:
should use css variables or theme file for colors so that it is easier to change later